website_lazy_load_image/models/ir_ui_view.py

```diff
@@ -22,7 +22,7 @@ def render_template(self, template, values=None, engine='ir.qweb'):
         website_id = self.env.context.get('website_id')
         if website_id and not \
                 self.env['website'].browse(website_id).is_publisher():
-            html = etree.HTML(res)
+            html = ehtml.fromstring(res.decode())
```

Comment:
hmm I don't understand: where is the check on whether this is or not an HTML source? :thinking:

Reply:
as you mentioned, lxml.html.fromstring keep the content as it's if it's not an html content.
I just tested it

Reply:
Also, I didn't face any problem with robots.txt as mentioned on the issue https://github.com/OCA/website/issues/551

Reply:
ok, cool! Can you update tests [here](https://github.com/OCA/website/blob/11.0/website_lazy_load_image/tests/test_lazy_load_image.py) to avoid regressions?